### PR TITLE
[PaddleDetection] add SSD config_demo

### PR DESCRIPTION
**Pull request by yghstill**
1. add SSD config_demo

### CODE REVIEW

File: PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+num_classes: 81
+
+# SSD architecture, see https://arxiv.org/abs/1512.02325
+SSD:
+  # backbone instance, defined below.
+  backbone: VGG
+  # `MultiBoxHead` instance, defined below.
+  multi_box_head: MultiBoxHead
+
+  # fluid.layers.detection_output, Detection Output Layer for SSD.
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml:38]: 需要简单描述下detection output的pipline

File: PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+  extra_block_filters:
+  - [256, 512, 1, 2, 3]
+  - [128, 256, 1, 2, 3]
+  - [128, 256, 0, 1, 3]
+  - [128, 256, 0, 1, 3]
+  # params list of init scale in l2 norm, skip init scale if param is -1.
+  normalizations: [20., -1, -1, -1, -1, -1]
+
+# fluid.layers.multi_box_head, Generate prior boxes for SSD algorithm.
+MultiBoxHead:
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml:82]: 同样需要简单描述下算法实现

File: PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+  # With forward computing and back propagation , Optimizer use back-propagation
+  # gradients to optimize parameters in a neural network.
+  optimizer:
+    # Momentum optimizer adds momentum on the basis of SGD ,
+    # reducing noise problem in the process of random gradient descent.
+    momentum: 0.9
+    type: Momentum
+  # fluid.regularizer
+  regularizer:
+    # mplements the L2 Weight Decay Regularization
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml]: implement?

File: PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml (hunk 1 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml:38]: DONE

File: PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml (hunk 2 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml:82]: DONE

File: PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml (hunk 3 above)
Comment (yghstill) [reply PaddleCV/PaddleDetection/docs/config_example/ssd_vgg16_300.yml]: DONE

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request merged**